src/bootstrap/config.toml.example

```diff
@@ -152,6 +152,9 @@
 # known-good version of OpenSSL, compile it, and link it to Cargo.
 #openssl-static = false
 
+# Disable the vendoring of libs when creating dist archives
+# disable-cargo-vendor = false
```

Comment:
I think this may not be the intent we wish to convey, perhaps "disable building the source tarball" ?